tools/scaffold.ts

```diff
@@ -28,13 +31,25 @@ await Promise.all([
 
 console.log(
   green("SUCCESS"),
-  `finished to scaffold for \`${kebabCasedLintName}\`!`,
+  `finished to scaffold for ${cyan(kebabCasedLintName)}!`,
 );
 console.log(
-  `Next, open \`docs/rules/${snakeCasedLintName}.md\` and \`src/rules/${snakeCasedLintName}.rs\` in your editor and implement the rule.`,
+  `Next, open ${cyan(`docs/rules/${snakeCasedLintName}.md`)} and ${
+    cyan(
+      `src/rules/${snakeCasedLintName}.rs`,
+    )
+  } in your editor and implement the rule.`,
 );
 console.log(
-  "Also, don't forget to manually add a new lint rule to `get_all_rules` function in `src/rules.rs` so that the rule will get to be run actually.",
+  `Also, don't forget to manually add a new lint rule to ${
+    cyan(
+      "get_all_rules",
+    )
+  } function in ${
+    cyan(
+      "src/rules.rs",
+    )
+  } so that the rule will get to be run actually.`,
 );
 
```

Comment:
Could you add some `Deno.test` here to check it works as expected? You'll probably need to gate using `import.meta.main` to run the actual script

Reply:
Done in 0c099c4, but running the script actually in the test isn't desirable as it has side effects (e.g. create and edit files), so I just added some unit tests that confirm case-conversion and the contents of generated files are correct.